api/schemas.py

```diff
@@ -30,7 +30,8 @@ def validate_schema(self, data, **kwargs):
 class StringPaginationSchema(ma.Schema):
     class Meta:
         ordered = True
-
+        
+    terms = ma.String()
```

Comment:
You are adding search terms to the pagination schema? That sounds a little strange, don't you think?